commons/passive/garmin/garmin_generic_stress.avsc

```diff
@@ -0,0 +1,24 @@
+{
+   "name":"GarminGenericStress",
+   "type":"record",
+   "namespace":"org.radarcns.passive.garmin",
+   "doc":"Provides the data about real time Stress data.",
```

Comment:
Is this about physical or mental stress? Please specify.

Reply:
There is no specific mention in Garmin docs if this stress is Physical or Mental. It is based on heart rate variability and this can be result of both mental and physical stress. 


From the doc - 
When using the stress level feature, the device uses heart rate data to determine the interval between each heart beat. The variable length of time in between each heart beat is regulated by the body's autonomic nervous system. The less variability between beats equals higher stress levels, whereas the increase in variability indicates less stress.  

https://support.garmin.com/en-US/?faq=WT9BmhjacO4ZpxbCc0EKn9

Reply:
Further this document clarifies that this stress score is about both Physical and Mental. Its paragraph is reproduced here - https://support.garmin.com/en-US/?faq=04pnPSBTYSAYL9FylZoUl5


There are many different statistical methods used to characterize HRV. The HRV stress test (formerly called stress score) feature puts your stress on an easy-to-understand 0-to-100 scale. This is specifically designed to be a tool for you to assess how your body is doing and how it’s handling training stress and life stress.

Reply:
Updated the description in document. available in latest checkin